build_tools/github_actions/fetch_test_configurations.py

```diff
@@ -111,6 +111,22 @@ def _get_script_path(script_name: str) -> str:
         "test_script": f"pytest {_get_script_path('test_rccl.py')} -v -s --log-cli-level=info",
         "platform": ["linux"],
     },
+    # FFT tests
+    "rocfft": {
```

Comment:
Sort these next to PRIM tests instead of adding after MIOpen/RCCL, given that both are in the "math-libs" component?
https://github.com/ROCm/TheRock/tree/main/math-libs